src/types/webhook.ts

```diff
@@ -228,3 +228,15 @@ export interface UpsertSlashCommandOptions {
   /** The parameters for the command */
   options?: SlashCommandOption[];
 }
+
+export interface UpsertSlashCommandsOptions {
+  commandID?: string;
+  /** 3-32 character command name */
+  name: string;
+  /** 1-100 character description */
+  description: string;
+  /** The parameters for the command */
+  options?: SlashCommandOption[];
+  /** @ignore this Parameter is used internally to make the request so you can safely ignore that. */
+  id?: string;
```

Comment:
If i understood this right, whats the purpose of commandID? Why not just use `id`

Reply:
To avoid ambiguity, I guess

Reply:
I want consistent typing. I have added this param so I don't need to use `@ts-ignore` but I can remove this param and add the comment if you want.